APIs/StreamCompatibilityManagementAPI.raml

```diff
@@ -275,7 +275,7 @@ mediaType: application/json
                 application/octet-stream:
                   type: file
             204:
-              description: There is no Base EDID set.
+              description: There is no Base EDID set or the Input doesn't support Base EDID or EDID at all.
```

Comment:
Does the use of `204` No Content and `405` Method Not Allowed when the Input doesn't support Base EDID or EDID at all seem a little inconsistent between methods of `/edid`, `/edid/base` and `/edid/effective`?

If EDID and Base EDID _are_ supported, then `204` No Content at `/edid/base` to indicate no Base EDID is set seems appropriate.

But the use of `204` to indicate EDID or Base EDID is not supported seems a bit odd.

The response to `/edid` normally indicates the sub-endpoints... if it returns `204` No Content, should they respond successfully? Also seems a bit strange to respond with a success code to `GET` `/edid/base` and `/edid/effective`, when `PUT` and `DELETE` are specified to return `405` Method Not Allowed to indicate the same thing...

Reply:
Even if it is preferable to keep `204` to indicate both not supported and supported-but-not-present on the `GET` requests for `/edid/base` and `/edid/effective`, would it still be a good idea to remove the `204` response from the parent `/edid` endpoint and always return `200` with the array of sub-endpoints?

Reply:
I agree to only support 200 to return the API hierarchy of end-points and possibly an extra 404 when there is an {id} in the path. (This is what we agreed in the meeting on Feb 20, 2023)